airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx

```diff
@@ -6,6 +6,8 @@ import * as yup from "yup";
 
 import { Label, Input, LoadingButton, LabeledToggle } from "components";
 import { Row, Cell } from "components/SimpleTableComponents";
+import { WebhookPayload } from "../../../../../components/hooks/services/useWorkspace";
```

Comment:
could you please remove relative imports? so they will start from `components` and `utils`